src/virtualenv/util/path/_sync.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import absolute_import, unicode_literals
+
+import logging
+import os
+import shutil
+from functools import partial
+
+import six
+
+HAS_SYMLINK = hasattr(os, "symlink")
```

Comment:
modern python3s on windows always have this attribute, just it's sometimes nonfunction (not sure it matters here)

Reply:
yeah, I know but they're not reliable; tested them a bit and even in my case had mixed results, so we don't use it for windows yet